client/public/index.html

```diff
@@ -124,6 +124,8 @@
           o &&
             ((document.documentElement.className = o),
             (document.documentElement.style.backgroundColor = c[o]));
+          const n = window.localStorage.getItem("nop");
```

Comment:
```suggestion
          const n = window.JSON.parse(sessionStorage?.whoami || "{}")?.settings?.noAds;
```

This should also work. It has the tiny caveat that it's only session storage. But that's a general issue. Wdyt?

Reply:
What is the advantage of your proposal?

Reply:
Ultimately, I disagree with the suggestion, because it couples the feature directly to an implementation detail of the whoami request caching, which is not a good practice and a recipe for bugs.